lib/engineering/analysis/footing/centric_isolated.rb

```diff
@@ -0,0 +1,48 @@
+module Engineering
+  module Analysis
+    module Footing
+      class CentricIsolated
+        ORTOGONALITIES = %i[length_x length_y].freeze
```

Comment:
I think the right english for this is `ORTHOGONALITIES`
```suggestion
        ORTHOGONALITIES = %i[length_x length_y].freeze
```